app/models/node.rb

```diff
@@ -71,7 +71,7 @@ def updated_month
     updated_at.strftime('%B %Y')
   end
 
-  has_many :revision, foreign_key: 'nid' # , dependent: :destroy # re-enable in Rails 5
```

Comment:
Hi @jywarren, I just found this `dependent : : destroy` commented due to which if a node of spammed revision is deleted by author or  moderator, the revisions aren't deleted. 

And, ` # , dependent: :destroy # re-enable in Rails 5` is present at many places so we have collecting redundant data lately, which is still present. 

Should we enable all `dependent: :destroy`?

Reply:
Hey @Souravirus any idea about these comments? Thanks!

Reply:
No, I didn't had any idea about these. Sorry about that.

Reply:
No issue!

Reply:
@jywarren these all comments are put by you at Rails 4.1 upgrade - https://github.com/publiclab/plots2/pull/1799/ 

Do you think we can remove these comments now? Also, should we delete the dependent data like revisions without node, etc?

Reply:
Hmm, i think we could open an issue for this - It was so long ago i don't remember. I think we have to be very very careful here, as incorrect `dependent: destroy` has caused data loss in the past. Let's do it separate from this PR! Thanks!

Reply:
![image](https://user-images.githubusercontent.com/24359/51000392-f8334580-14fa-11e9-877c-d5485472036f.png)